exercises/concept/lasagna/.docs/instructions.md

```diff
@@ -1,17 +1,14 @@
 # Instructions
 
-In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+Lucian's girlfriend is on her way home and he hasn't cooked their anniversary dinner!
+
+In this exercise, you're going to write some code to help Lucian cook an exquisite lasagna from his favorite cook book.
 
 You have four tasks, all related to the time spent cooking the lasagna.
 
 ## 1. Define the expected oven time in minutes
 
-Define the `EXPECTED_MINUTES_IN_OVEN` constant that returns how many minutes the lasagna should be in the oven. It must be exported. According to the cooking book, the expected oven time in minutes is 40:
-
-```javascript
-EXPECTED_MINUTES_IN_OVEN;
-// => 40
-```
```

Comment:
I am proposing in the PR to remove this example as the student might be confused whether line 13 means there is a return value from a function. I think the instructions are clearer without that part.

Reply:
Oki!